internal/pkg/source/testdata/src/sanitization/test.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sanitization
+
+import "fmt"
+
+type foo struct {
+	name     string
+	password string
+}
+
+func f1() {
+	foo := &foo{name: "bar", password: "P@ssword01"}
```

Comment:
Might be best to avoid name-showing unless we're explicitly testing for it.

Reply:
Replaced with simpler values.